pystiche_papers/johnson_alahi_li_2016/_modules.py

```diff
@@ -262,6 +289,7 @@ def __init__(
             self.init_weights()
 
     def init_weights(self) -> None:
+        r"""Initializes the weights of the transformer as it is the case in the original implementation."""
```

Comment:
Please describe the initialization. Should be one of "Kaiming" or "He", but I'm not sure.